pyproject.toml

```
@@ -71,6 +71,7 @@ x = "awx_plugins.credentials.x.api:XPlugin"
 [project.entry-points."awx.credential_plugins"]  # pre-existing entry points group name
 x = "awx_plugins.credentials.x.api:XPlugin"
 # conjur = awx.main.credential_plugins.conjur:conjur_plugin
+"aws_assumerole = awx_plugins.credentials.aws_assumerole:aws_assumerole_plugin"
```

Still a syntax error. The previous suggestion was correct:

```suggestion
aws_assumerole = "awx_plugins.credentials.aws_assumerole:aws_assumerole_plugin"
```

Just click the `Commit suggestion` button on GH UI next time. It'll make a proper commit, additionally preserving the credit.

---

ack

---

@thedoubl3j did you accidentally remove the entry point?

---

@webknjaz yup, started rebasing and getting pulled multiple dirs. patch inc